packages/ketcher-react/src/script/editor/utils/selectItems/startSelecting.ts

```diff
@@ -0,0 +1,137 @@
+import { FunctionalGroup, SGroup } from 'ketcher-core'
```

Comment:
feature related changes in this file -> mousedown logic based on `select` instrument without lasso